### PR TITLE
Support both polkadot and kusama runtimes

### PR DESCRIPTION
**Pull request by arkpar**
This adds support for two native runtimes in the `polkadot ` executable: Polkadot and Kusama. The CLI `--chain` param now selects native runtime as well.
`polkadot-runtime-common` crate is introduced which contains runtime code shared between both runtimes.

Since Kusama is supposed to be testbed for Polkadot we expect the two runtimes not to diverge much over time.

After this is merged `runtime/polkadot/src/lib.rs` and `runtime/polkadot/src/constants.rs` should be reviewed and updated with Polkadot values.

### CODE REVIEW

File: validation/Cargo.toml [hunk 1]
```diff
@@ -18,7 +18,7 @@ availability_store = { package = "polkadot-availability-store", path = "../avail
 parachain = { package = "polkadot-parachain", path = "../parachain" }
 polkadot-primitives = { path = "../primitives" }
 polkadot-erasure-coding = { path = "../erasure-coding" }
-polkadot-runtime = { path = "../runtime" }
+polkadot-runtime = { path = "../runtime/polkadot" }
```
Comment (bkchr) [inline validation/Cargo.toml]: Internally the runtime is used to get the `NEW_HEADS_INDENTIFIER`. This identifier should be moved to primitives and the reference to the runtime should be removed here.

File: runtime/polkadot/src/constants.rs [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+	pub const DOTS: Balance = 1_000_000_000_000;
+	pub const DOLLARS: Balance = DOTS / 100;
+	pub const CENTS: Balance = DOLLARS / 100;
+	pub const MILLICENTS: Balance = CENTS / 1_000;
+}
+
+/// Time and blocks.
+pub mod time {
+	use primitives::{Moment, BlockNumber};
+	// Kusama & mainnet
```
[the remark this answers is not in the document]
Comment (arkpar) [reply runtime/polkadot/src/constants.rs]: They already are. This is polkadot-specific runtime.

[the remark this answers is not in the document]
Comment (bkchr) [reply runtime/polkadot/src/constants.rs]: Okay, than you should remove the Kusama mentions in the comments.

[the remark this answers is not in the document]
Comment (arkpar) [reply runtime/polkadot/src/constants.rs]: I specifically mentioned in the PR description that these values should be updated along with `runtime/polkadot/src/lib.rs` with mainnet values. Probably by @gavofyork or yourself. I'm not confident enough with how the runtimes should be different. 

For now I'll set epoch duration to 4 hours, as this is what the comment suggests.

File: scripts/gitlab/check_runtime.sh [hunk 3]
```diff
@@ -47,74 +45,79 @@ then
...
+			changes to the runtime sources and changes in the impl version.
 
-		impl_version: ${sub_impl_version} -> ${add_impl_version}
+			impl_version: ${sub_impl_version} -> ${add_impl_version}
 
-		EOT
-		exit 0
-	fi
+			EOT
+			exit 0
```
Comment (gabreal) [inline scripts/gitlab/check_runtime.sh]: LGTM in general, but shouldn't this also be `return`?
made also an [update](https://github.com/paritytech/polkadot/tree/ci-a-runtimes) but your is also good to go with for me.

Comment (arkpar) [reply scripts/gitlab/check_runtime.sh]: Merged in your commit

**Pull request merged**